app/presenters/change_history_presenter.rb

```diff
@@ -27,7 +27,6 @@ def presented_change_notes
 
     def change_notes
       ChangeNote
-        .where(document: document)
         .where("edition_id IS NULL OR edition_id IN (?)", edition_ids)
```

Comment:
I assume we could also do this through one query with joins and skip the edition_ids query?